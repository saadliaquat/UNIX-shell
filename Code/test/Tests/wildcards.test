ls *
ls *.c
echo ../src/*.c