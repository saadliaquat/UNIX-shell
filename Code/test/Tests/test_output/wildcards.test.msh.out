config.json
config.json.log
find.log
names.txt
test.py

Tests:
aliases.test
aliases.test.custom
builtins.test
chaining.test
ioredir.test
pipeline.test
quotes.test
simple.test
test_output
wild_chaining.test
wildcards.test
../src/main.c
